segmentify/semantic/main.py

```diff
@@ -16,15 +16,17 @@ def _load_model(pretrained_model):
     """
     # get pretraineod model saved filed path
     ## TODO better way to store and define file path
+    # Use relative path
+    # Use os join
     if pretrained_model == "HPA":
-        model_path = "/home/mars/CZI/segmentify/segmentify/model/saved_model/UNet_hpa_max.pth"
+        model_path = "/home/mars/CZI/segmentify/segmentify/model/saved_model/UNet_hpa_protein.pth"
```

Comment:
can you fix this path in this PR too - seems like fixing all the paths belongs in one PR rather than lots of small ones, but either way is fine

Reply:
When I used the relative path
```model_path = os.path.join("..","model","saved_model","UNet_hpa_max.pth")```
I get this error when running the example:
"FileNotFoundError: [Errno 2] No such file or directory: '../model/saved_model/UNet_hpa_max.pth'"

Reply:
Ah ok - you may need to get the current path of the file then. @kne42 will likely know the best way to do this in python, maybe she can advise

Reply:
Ah ok - you may need to get the current path of the file then. @kne42 will likely know the best way to do this in python, maybe she can advise

Reply:
Ah ok - you may need to get the current path of the file then. @kne42 will likely know the best way to do this in python, maybe she can advise

Reply:
The path only works when I set it to:
```model_path = os.path.join(".","segmentify","model","saved_model","UNet_hpa_max.pth")```
which means that the path is relative to where the user is running the code.

Reply:
Yeah, so you can ask python to give you the absolute path of the current file you are in “XYZ/segmentify/semantic/main.py” and you can then parse this and join the relative path you have to get the correct absolute path without having to hard code it

Reply:
Yeah, so you can ask python to give you the absolute path of the current file you are in “XYZ/segmentify/semantic/main.py” and you can then parse this and join the relative path you have to get the correct absolute path without having to hard code it